### PR TITLE
Cards 18 - 20

### PR DESCRIPTION
**Pull request by cotoole-merkle**

### CODE REVIEW

File: .env
Comment (JRondeau16) [inline .env:1]: Can revert this file so it's not part of the commit?

File: scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml
Comment (JRondeau16) [inline scaffolding/merkle.feature.basiccontent/src/Feature/BasicContent/items/Templates/Basic Content/ContentPodGroup.yml:1]: The content pod group itself is the container that contains a number of content pods, so it doesn't need a template. This template looks good for the content pod, though so it just needs to be renamed.

**Review by JRondeau16 — CHANGES_REQUESTED** [review]

**Comment by cheeto-bandito** [comment]
Two key things:

1. The .env file should not be changed.  There are secrets in here that are now part of git history.
2. The _Title and _Description base templates can be re-used across all three components.  Consider creating a standard datasource template for all three?
